src/Task/Psalm/PsalmTask.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Wunderio\GrumPHP\Task\Psalm;
+
+use GrumPHP\Collection\ProcessArgumentsCollection;
+use Wunderio\GrumPHP\Task\AbstractMultiPathProcessingTask;
+
+/**
+ * Class PsalmTask.
+ *
+ * Psalm task.
+ *
+ * @package Wunderio\GrumPHP\Task
+ */
+class PsalmTask extends AbstractMultiPathProcessingTask {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function buildArguments(iterable $files): ProcessArgumentsCollection {
+    $arguments = $this->processBuilder->createArgumentsForCommand('psalm');
+    $config = $this->getConfig()->getOptions();
+    $arguments->addOptionalArgument('--output-format=%s', $config['output_format']);
+    $arguments->addOptionalArgument('--config=%s', $config['config']);
+    $arguments->addOptionalArgument('--report=%s', $config['report']);
+    $arguments->addOptionalArgument('--no-cache', $config['no_cache']);
+    $arguments->addOptionalArgument('--threads=%d', $config['threads']);
+    $arguments->add('--find-unused-code');
```

Comment:
When testing on one of client's projects, `UnusedVariable` errors can't be ignored if using `/** @psalm-suppress UnusedVariable */` annotation-comment before the variable., as `--find-unused-code` seems to have higher priority. It can be changed by ignoring particular errors in psalm.xml file by adding
```
<issueHandlers>
    <UnusedVariable errorLevel="info" />
</issueHandlers>
``` 
but prolly all new `UnusedVariable` errors will be ignored from now on, which is not great, I guess.